Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift

```diff
@@ -17,18 +17,27 @@
 import SwiftKuery
 import CLibpq
 
+import Dispatch
 import Foundation
 
+enum ConnectionState {
```

Comment:
This might be better as a nested enum inside `PostgreSQLConnection`, called `State`.